src/lib/pickles/wrap_verifier.ml

```diff
@@ -730,7 +756,8 @@ struct
   let incrementally_verify_proof (type b)
       (module Max_proofs_verified : Nat.Add.Intf with type n = b)
       ~actual_proofs_verified_mask ~step_domains ~srs
-      ~verification_key:(m : _ Plonk_verification_key_evals.Step.t) ~xi ~sponge
+      ~verification_key:(m : (_ array, _) Plonk_verification_key_evals.Step.t)
```

Comment:
maybe let's add some type var name explicitely describing the types (commitments)? Just for readability.

Reply:
This feels orthogonal to this PR. This is only here because I used it to help give clear type errors, but happy to revert it.